docs/user-guide.md

```diff
@@ -24,26 +24,26 @@ that might be null then you write `@Nullable String x;`.
 
 ## Types and nullness
 
-In the JSpecify world, most references have one of three possible properties
+In the JSpecify world, most TODO references have one of three possible properties
```

Comment:
The `most` is never explained. Maybe a forward reference here, or a backward reference from generics that explains the "most" aspect would be nice.

Reply:
Yeah... saying anything about the exceptions would be a harmful distraction right now. Can we say "all" but then "(We'll cover a few exceptions later.)"?